### DISCUSSION TITLE
求助：给P2P模式主节点添加node工作节点

### DISCUSSION THREAD

**Discussion by xingchl** [secretflow/secretflow#1657] [General]
P2P模式下，给主节点添加node工作节点。我理解的主节点是k3s的控制节点，也是工作节点，怎么能给他再添加一个工作节点。

**Comment by UniqueMarvin** [comment]
理论上可以将 kuscia Autonomy 当 kuscia master 使用的，即可以调用 Autonomy 的 KusciaAPI Create Domain ，然后部署lite 节点，将新部署的 lite 节点注册至 Autonomy节点 。
但这里要看一下实际的使用场景，如果某一方（alice）存在需要使用多个节点的场景，那建议部署 master+lite ， 另一方（bob）可以部署 Autonomy 或者  master+lite 。

**Comment by Monesph** [comment]
我也有类似的需求，我尝试部署domainid相同的多个lite，但是只有最后部署的那个lite节点是ready状态，请问您这边问题解决了吗？
